runtime/common/src/precompile/schedule.rs

```diff
@@ -489,7 +489,7 @@ mod tests {
 			run_to_block(5);
 			#[cfg(not(feature = "with-ethereum-compatibility"))]
 			{
-				assert_eq!(Balances::free_balance(from_account.clone()), 999999944068);
+				assert_eq!(Balances::free_balance(from_account.clone()), 999999944095);
```

Comment:
not 100% sure why this number changed

Reply:
we need to update tests to make those values calculated or add a good explanation why. We've a lot of tests with hardcoded values and it's very hard to judge why they change